plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -221,7 +222,12 @@ public ConnectorTableHandle getTableHandle(ConnectorSession session, SchemaTable
             return null;
         }
 
-        return new BigQueryTableHandle(schemaTableName, new RemoteTableName(tableInfo.get().getTableId()), tableInfo.get());
+        return new BigQueryTableHandle(new BigQueryNamedRelationHandle(
+                schemaTableName,
+                new RemoteTableName(tableInfo.get().getTableId()),
+                tableInfo.get().getDefinition().getType().toString(),
+                getPartitionType(tableInfo.get().getDefinition()),
+                Optional.ofNullable(tableInfo.get().getDescription())));
```

Comment:
This commit adds more then just relation handle i.e. it adds support for table comments.

Reply:
Table comment is already supported in BigQuery connector https://github.com/trinodb/trino/blob/35f68312ed7dc065245d4d55d9420e10d74069f8/plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTableHandle.java#L54